### ISSUE TITLE
Components not linted if no immediate rule

### ISSUE THREAD

**Issue by AlecRust** [postcss/postcss-bem-linter#134]
If a selector doesn't have an immediate rule the block doesn't get linted e.g.

```css
/** @define Test */

.Test {}

.anything {

  @media (--viewport-medium) {
    width: 100%;
  }
}
```

`.anything` doesn't get linted, but it does if this is the CSS:

```css
/** @define Test */

.Test {}

.anything {
  color: red;

  @media (--viewport-medium) {
    width: 100%;
  }
}
```

**Comment by simonsmith** [comment]
Related to #119 - Need to adjust that solution to support `@media`

**Comment by simonsmith** [comment]
Fixed in #140
